roles/kubernetes-apps/csi_driver/aws_ebs/templates/aws-ebs-csi-controllerservice-rbac.yml.j2

```diff
@@ -58,16 +58,16 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "update"]
-  - apiGroups: [""]
-    resources: ["nodes"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["csi.storage.k8s.io"]
-    resources: ["csinodeinfos"]
+    verbs: ["get", "list", "watch", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csinodes"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "update"]
+    verbs: ["get", "list", "watch", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments/status"]
+    verbs: ["patch"]
```

Comment:
Where did this change come from?
I cannot find the reason on both this PR message and the corresponding issue.
In addition, the same name of ClusterRole of https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/aws-ebs-csi-driver/templates/clusterrole-attacher.yaml is the same as the original one, and different from this PR.

Reply:
The reason for this change is in https://github.com/kubernetes-sigs/kubespray/issues/6770#issuecomment-701391883. The version of `csi-attacher` was bumped from `v1.2.1` to `v2.2.0` in Kubespray 2.14.0 for CSI drivers that do not use additional variable for the `csi-attacher` image tag (that is why someone else had problems with Cinder CSI and fixed them in https://github.com/kubernetes-sigs/kubespray/pull/6358). The new version of `csi-attacher` requires different [permissions](https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/rbac.yaml#L23) to function properly. Without this change the volumes could not be attached to pods and the `csi-attacher` container had lots of errors in logs related to permissions.

Also, the file linked by you is for `v1.2.0` of `csi-attacher` (https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/aws-ebs-csi-driver/values.yaml#L18) and that is why it is the same as the original one.

Reply:
Thanks for your explanation, I got it.
How about putting comment like `# This ClusterRole comes external-attacher-runner of kubernetes-csi/external-attacher` or something?
It can solve questions like mine and avoid reverting this change.

Reply:
I guess that I would prefer adding some kind of test for it but my knowledge about kubespray's CI is very limited so let's go with the comment. How about:
```yaml
# The permissions in this ClusterRole are tightly coupled with the version of csi-attacher used. More information about this can be found in kubernetes-csi/external-attacher.
```
?

Reply:
Thanks for your suggestion, that looks good for me :-)

Reply:
Done ;)